test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -194,7 +194,8 @@ public static final ProtocolVersion getDefaultVersion()
         }
     }
 
-    public static ResultMessage lastSchemaChangeResult;
+    public static volatile ResultMessage lastSchemaChangeResult;
```

Comment:
can you update the usages to pull this reference once?  for example `org.apache.cassandra.cql3.CQLTester#assertLastSchemaChange` access it twice, which isn't safe.